monalisa.sh

```diff
@@ -0,0 +1,37 @@
+package: MonALISA
+version: "20210818"
+# tag: "1.0.0"
+# source: https://gitlab.cern.ch/jalien/jalien.git
+requires:
+ - JDK
+ - "curl:(?!slc8)"
+ - "system-curl:slc8.*"
+valid_defaults:
+ - monalisa
```

Comment:
Please, do not introduce another default.

Reply:
Sorry, this looks weird, I will have to look up how to properly use github suggestions, I have not committed a suggestion before.

Reply:
somehow your comment doesn't show as a suggestion and mine is little messed up :)